src/Libraries/DynamoUnits/Utilities.cs

```diff
@@ -404,5 +404,129 @@ internal static bool TryParseTypeId(string typeId, out string typeName, out Vers
 
             return false;
         }
+
```

Comment:
As shown here, `AddAscSchemaPaths` uses reflection to obtain `AscSdkWrapper` from `DynamoInstallDetective.dll`. This approach avoids introducing a build-time dependency on the DLL, which is Windows-only, while `DynamoUnits` remains platform-agnostic.